paddle/fluid/inference/api/demo_ci/CMakeLists.txt

```diff
@@ -139,6 +140,7 @@ if(WITH_GPU)
       set(DEPS ${DEPS} ${TENSORRT_LIB_DIR}/libnvinfer_plugin${CMAKE_STATIC_LIBRARY_SUFFIX})
     endif()
     set(DEPS ${DEPS} ${CUDA_LIB}/libcudart${CMAKE_SHARED_LIBRARY_SUFFIX})
+    set(DEPS ${DEPS} ${PADDLE_LIB}/third_party/install/dgc/lib/libdgc.so)
```

Comment:
inference should not depend dgc since it's only for training

Reply:
Done.